LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -1,13 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="MSBuild.Sdk.Extras/1.6.46">
   <PropertyGroup>
     <Title>LibVLCSharp</Title>
     <Summary>.NET bindings for LibVLC</Summary>
     <Description>.NET bindings for LibVLC</Description>
     <PackageTags>libvlc native xamarin .net video audio media mediaplayer</PackageTags>
-    <TargetFrameworks>netstandard2.0;MonoAndroid81;net461;netcoreapp2.0;Xamarin.iOS10;Xamarin.Mac20</TargetFrameworks>
```

Comment:
`net461` is a target in the nuget. If you remove that, I don't know what happens to packaging and whether netstandard2.0 is enough.
Same for `netcoreapp2.0`, is `netstandard2.0` enough for all .NET Core apps?

I'm really not sure... most of the Xamarin projects I see (reactiveUI, mvvmcross) use all the targets.

Reply:
Do you have a good reason to want to remove those target framework? (aside from the "whats useless is bad" argument)

Reply:
- LibVlcSharp package weight : When you download it, the whole package with all platforms is downloaded. With 1 platform instead of 3 for desktop, you avoid duplicates
- Less combination /complexity in LibVlcSharp project : Your `#if` make code variants and the fewer variants, the better the code is maintainable
- .NET standard's first reason is to make libraries that works on .net framework and on .net core. I think the main benefit from that is that you can avoid multi targetting code that can be standardized.

Reply:
Fair enough. Will test to make sure that doesn't break anything unexpectedly.